<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-15S6U3P" failures="10" tests="11" name="Test1(failed)" time="110.165" errors="0" timestamp="2023-01-11T03:44:39 EST">
  <testcase classname="testLayer.RegisterTest" name="Register_invalidwithNumber" time="8.01">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testLayer.RegisterTest.Register_invalidwithNumber(RegisterTest.java:113)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Register_invalidwithNumber -->
  <testcase classname="testLayer.RegisterTest" name="Register_invalidwithSpecialCharacter" time="5.315">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testLayer.RegisterTest.Register_invalidwithSpecialCharacter(RegisterTest.java:136)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Register_invalidwithSpecialCharacter -->
  <testcase classname="testLayer.RegisterTest" name="Register_invalidwithSpace" time="4.872">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testLayer.RegisterTest.Register_invalidwithSpace(RegisterTest.java:159)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Register_invalidwithSpace -->
  <testcase classname="testLayer.RegisterTest" name="Register_invalidEmpty" time="5.79">
    <failure type="java.lang.AssertionError" message="Arrays differ at element [6]: Phone number is required. !=  expected [Phone number is required.] but found []">
      <![CDATA[java.lang.AssertionError: Arrays differ at element [6]: Phone number is required. !=  expected [Phone number is required.] but found []
at testLayer.RegisterTest.Register_invalidEmpty(RegisterTest.java:170)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- Register_invalidEmpty -->
  <testcase classname="testLayer.RegisterTest" name="Register_invalidPin" time="5.52">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testLayer.RegisterTest.Register_invalidPin(RegisterTest.java:189)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Register_invalidPin -->
  <testcase classname="testLayer.RegisterTest" name="Register_invalidPin" time="5.465">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testLayer.RegisterTest.Register_invalidPin(RegisterTest.java:189)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Register_invalidPin -->
  <testcase classname="testLayer.RegisterTest" name="Register_invalidPhoneno" time="5.258">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testLayer.RegisterTest.Register_invalidPhoneno(RegisterTest.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Register_invalidPhoneno -->
  <testcase classname="testLayer.RegisterTest" name="Register_invalidPhoneno" time="5.257">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testLayer.RegisterTest.Register_invalidPhoneno(RegisterTest.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Register_invalidPhoneno -->
  <testcase classname="testLayer.RegisterTest" name="Register_invalidSSn" time="5.024">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testLayer.RegisterTest.Register_invalidSSn(RegisterTest.java:233)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Register_invalidSSn -->
  <testcase classname="testLayer.RegisterTest" name="Register_invalidSSn" time="5.528">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testLayer.RegisterTest.Register_invalidSSn(RegisterTest.java:233)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- Register_invalidSSn -->
  <testcase classname="testLayer.RegisterTest" name="Register_invalidConfirmpass" time="7.759"/>
  <testcase classname="testLayer.RegisterTest" name="Register_valid" time="0.0">
    <ignored/>
  </testcase> <!-- Register_valid -->
</testsuite> <!-- Test1(failed) -->
